api/src/main/java/net/kyori/adventure/text/format/TextColor.java

```diff
@@ -188,6 +188,35 @@
     return this.value() & 0xff;
   }
 
+  /**
+   * Linearly interpolates between {@code this} and {@code that} by {@code t}.
```

Comment:
Can there be a `<p>` that explains what this does in layman's terms? I don't think the average user of adventure has a solid grasp of linear interpolation with respect to colors.

e.g. https://p5js.org/reference/#/p5/lerpColor